yarn-project/validator-client/src/validator.ts

```diff
@@ -94,9 +124,17 @@ export class ValidatorClient extends WithTracer implements Validator {
     });
     try {
       await this.ensureTransactionsAreAvailable(proposal);
+
+      if (this.config.validatorReEx) {
+        this.log.verbose(`Re-executing transactions in the proposal before attesting`);
+        await this.reExecuteTransactions(proposal);
+      }
     } catch (error: any) {
       if (error instanceof TransactionsNotAvailableError) {
         this.log.error(`Transactions not available, skipping attestation ${error.message}`);
+      } else {
```

Comment:
Is it worth logging if re-execution failed?

Reply:
good shout

Reply:
ah, it already does, when the ReexecutionStateMismatch is the error, it will output that in the current log